examples/predictions.ipynb

```diff
@@ -187,257 +269,339 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "To estimate theoretical uncertainties of the observables, EOS uses Bayesian statistics.\n",
-    "The latter interprets the theory parameters as random variables and assigns *a priori*\n",
-    "probability density functions (prior PDFs) for each parameter.\n",
+    "To estimate theoretical uncertainties of an observable $O$, EOS uses Bayesian statistics.\n",
+    "The latter interprets the theory parameters as random variables $\\vec\\vartheta$ and assigns *a priori*\n",
```

Comment:
What do you mean by "the latter" here? Did this previous sentence originally contain two things (like Bayesian not frequentist, or something) at some point?

Reply:
No, this is a screw up by a non-native speaker. Will fix it!